biogtr/models/global_tracking_transformer.py

```diff
@@ -19,15 +19,13 @@ def __init__(
         nhead: int = 8,
         num_encoder_layers: int = 6,
         num_decoder_layers: int = 6,
-        dim_feedforward: int = 1024,
         dropout: int = 0.1,
         activation: str = "relu",
         return_intermediate_dec: bool = False,
-        feature_dim_attn_head: int = 1024,
         norm: bool = False,
         num_layers_attn_head: int = 2,
         dropout_attn_head: int = 0.1,
-        embedding_meta: dict = None,
+        embedding_meta: dict = {"pos": None, "temp": None},
```

Comment:
No mutables as defaults! Set it to `None` and initialize it as the dictionary inside the constructor. Same for `encoder_cfg`. If you change those objects after instance creation, they'll change the defaults for all future instances of the class.